.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Node.js CI
 
-on: [push, pull_request]
```

Comment:
The downside of this change is that when someone contributes externally, we do not get CI builds run anymore.

Reply:
Yes, you're right! I didn't consider that use-case, and will add it back